uportal-war/src/main/java/org/jasig/portal/io/xml/SpELDataTemplatingStrategy.java

```diff
@@ -80,11 +79,11 @@ public Source processTemplates(Document data, String filename) {
                         }
                         break;
                     case org.w3c.dom.Node.TEXT_NODE:
-                        Text t = (Text) n;
-                        inpt = t.getText();
```

Comment:
Gary is TEXT_NODE a subset of CDATA_SECTION_NODE?  E.g. will a text node get processed within the CDATA_SECTION_NODE case?